.github/actions/verify-helm/aps_helm_install.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
```

Comment:
I think this file is just an outdated version of acs_helm_install, try to get rid of this one

Reply:
For now I use the helm_install.sh script form acs-deployment for both, but  I needed to add a check to skip some tests which were in the acs version but not the aps one, because they were failing. For now  both should work the same as before migrating the actions to this repo.